src/python_be.cc

```diff
@@ -2041,6 +2041,8 @@ ModelState::LaunchAutoCompleteStubProcess()
         ex.err_ == nullptr, TRITONSERVER_ERROR_INTERNAL,
         std::string("unexpected nullptr in BackendModelException"));
     RETURN_IF_ERROR(ex.err_);
+    return TRITONSERVER_ErrorNew(
```

Comment:
Trying to understand here. Shouldn't the RETURN_IF_ERROR call in L2043 detect ex.err_ is not nullptr and returned the error object?

Reply:
+1, I had the same thought

Reply:
Hi @rmccorm4 and @tanmayv25,
Initially my assumption was RETURN_IF_ERROR(ex.err_); is not returning for some reason hence I added a ruturn statement manually.
This seemed to fix the issue.

Later turned out the way I was reproducing the issue was inconsistent and this statement has no impact on the bug :)
After spending quite some time, I was able to consistently reproduce.

Steps to repro

1. Load a "correct" model succesfully.
2. Try loading a "in-correct/faulty" model after step 1. Triton will hang.

If we directly try to  
Load a "in-correct/faulty" model Triton erros out and returns 400 (expected and correct behavior)

Reply:
@indrajit96 Thanks for the details! Are you able to get the backtrace and see which line causes the hang? Is the hang still related to the auto-complete stub after the new repro? If disabling the auto-complete, will the hang still occur?

Reply:
Resolved this thread after discussion with @krishung5 and also reverted that chunk of code